senza/manaus/utils.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, Optional  # noqa: F401
+
+from botocore.exceptions import ClientError
+
+__all__ = ["extract_client_error_code"]
```

Comment:
We highly encourage to not do `from foobar import *` even flake8 [will complain](http://flake8.pycqa.org/en/latest/user/error-codes.html) about that. So I guess we do not need to define `__all__` here.